neomodel/match.py

```diff
@@ -7,13 +7,24 @@
 
 def rel_helper(**rel):
     if rel['direction'] == OUTGOING:
-        stmt = '-[{0}:{1}]->'
+        stmt = '-{0}->'
     elif rel['direction'] == INCOMING:
-        stmt = '<-[{0}:{1}]-'
+        stmt = '<-{0}-'
     else:
-        stmt = '-[{0}:{1}]-'
-    ident = rel['ident'] if 'ident' in rel else ''
-    stmt = stmt.format(ident, rel['relation_type'])
+        stmt = '-{0}-'
```

Comment:
I think defaulting to `(x)--(y)` is okay since there isn't a performance issue and the user will still be able to traverse in either direction they'd like. I guess I'm asking if we need to define it as 

    if rel['direction'] == OUTGOING:
        stmt = '-{0}->'
    elif rel['direction'] == INCOMING:
        stmt = '<-{0}-'
    else:
        stmt = '-{0}-'

instead of removing the need for OUTGOING and INGOING and simply define it as

    stmt = '-{0}-'

Since neo4j doesn't seem to care internally about direction. Unless I'm reading 

```
Relationships in Neo4j can be traversed in both directions with the same speed. Moreover, 
direction can be completely ignored. Therefore, there is no need to create two different 
relationships between nodes, if one implies the other.
```
incorrectly. Do you know if using `(x)--(y)` causes unexpected behavior when used in place of `(x)<--(y)' or `(x)-->(y)`?